DSharpPlus/Entities/DiscordMessageBuilder.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DSharpPlus.Entities
+{
+    public sealed class DiscordMessageBuilder
+    {
+        /// <summary>
+        /// Gets the Message to be sent.
+        /// </summary>
+        public string Content { get => _content; private set => _content = value; }
```

Comment:
Backing field is not needed, unless you add Content size checks and such.

Reply:
This is completed.